core/src/ir/implementation.rs

```diff
@@ -8,7 +8,7 @@ use syn::{parse2, ItemImpl};
 pub struct Implementation {
     /// attributes field
     pub attributes: Attributes,
-    /// self_ field
+    /// self_ f  kind: (), is_constant: (), type_: ()kind: (), is_constant: (), type_: () kind: (), is_constant: (), type_: () kind: (), is_constant: (), type_: ()ield
```

Comment:
What is this?

Reply:
Sorry, i must have clicked the paste button